bika/lims/content/sampletype.py

```diff
@@ -198,6 +254,158 @@ def ContainerTypesVocabulary(self):
         from bika.lims.content.containertype import ContainerTypes
         return ContainerTypes(self, allow_blank=True)
 
+    def getDefaultSmallSticker(self):
+        """
+        Returns the small sticker ID defined as default.
+
+        :return: A string as an sticker ID
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return ''
+        value = values[0].get('small_default')
+        return value
+
+    def getDefaultLargeSticker(self):
+        """
+        Returns the large sticker ID defined as default.
+
+        :return: A string as an sticker ID
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return ''
+        value = values[0].get('large_default')
+        return value
+
+    def getAdmittedStickers(self):
+        """
+        Returns the admitted sticker IDs defined.
+
+        :return: An array of sticker IDs
+        """
+        values = self.getField('AdmittedStickerTemplates').get(self)
+        if not values:
+            return []
+        admitted = values[0].get('admitted')
+        return admitted
+
+    def _small_default_voc(self):
+        """
+        Returns the vocabulary to be used in
+        AdmittedStickerTemplates.small_default
+
+        If the object has saved not AdmittedStickerTemplates.admitted stickers,
+        this method will return an empty DisplayList. Otherwise it returns
+        the stickers selected in admitted.
+
+        :return: A DisplayList
+        """
+        admitted = self.getAdmittedStickers()
+        if not admitted:
+            return DisplayList()
+        voc = DisplayList()
+        stickers = getStickerTemplates()
+        for sticker in stickers:
+            if sticker.get('id') in admitted:
+                voc.add(sticker.get('id'), sticker.get('title'))
+        if voc.index > 0:
+            return voc
+        return DisplayList()
+
+    def _large_default_voc(self):
+        """
+        Returns the vocabulary to be used in
+        AdmittedStickerTemplates.large_default
+
+        If the object has saved not AdmittedStickerTemplates.admitted stickers,
+        this method will return an empty DisplayList. Otherwise it returns
+        the stickers selected in admitted.
+
+        :return: A DisplayList
+        """
+        admitted = self.getAdmittedStickers()
+        if not admitted:
+            return DisplayList()
+        voc = DisplayList()
+        stickers = getStickerTemplates()
+        for sticker in stickers:
+            if sticker.get('id') in admitted:
+                voc.add(sticker.get('id'), sticker.get('title'))
+        if voc.index > 0:
+            return voc
+        return DisplayList()
```

Comment:
Same exact function as `_small_default_voc`. Please use just only one function `get_sticker_templates_vocabularies()` as suggested in a comment above